setup.cfg

```diff
@@ -0,0 +1,25 @@
+[bumpversion]
+current_version = "4.3.0.dev"
+commit = False
+tag = False
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+))?((?P<build>\d+))?
+serialize =
+	{major}.{minor}.{patch}.{release}{build}
+	{major}.{minor}.{patch}.{release}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+optional_value = stable
+first_value = stable
+values =
+    stable
+    dev
+    a
+    b
+
+[bumpversion:part:build]
+first_value = 0
+
+[bumpversion:file:setup.py]
```

Comment:
Right now the only place where we have version explicitly set is 
https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/VERSION

Can bump2version manage that static file instead of putting version back to setup.py and `__init__.py` ?

Reply:
yeah, feels like bump2version should update VERSION unless there is a reason to add it back in the other 2 files.

bump2version will really shine when the UI gets absorbed into this repo and there will be additional version strings scattered around

Reply:
And if we go with putting versions back on .py files, then VERSION file must be deleted and removed from MANIFEST.in

Reply:
Using galaxy_ng/app/VERSION requires setup.py and galaxy_ng/app/__init__.py to read the version number at runtime. If we make those versions static with bump2version, that's two less runtime couplings that can break.

If we want to move more of the setup.py data into setup.cfg[1] (seems to be currently suggested approach by the docs), then setup.cfg can't run python code to read in the version.

[1] or pyproject.toml or whatever

Reply:
@alikins 

good points

1) setup.py already needs to read some static files and also find packages, so there is no problem to have it loading one more.

2) for `__init__`  you have a good point, we can change the code there, as setup.py outputs the version already read and parsed we can do.

```py
import pkg_resources
pkg_resources.get_distribution("galaxy_ng").version
```

Reply:
@rochacbruno do any of the pulp bits require a `galaxy_ng/app/VERSION` be present?

Reply:
@awcrosby no, pulp doesn't require it, pulp only requires version to be defined in the `AppConfig` class. VERSION was included just to ensure we have version written in a single place and that it allows us to read it from anywhere even if package is not installed.

We need to read the version in `setup.py` in `/galaxy_ng/app/__init__.py` and in `galaxy_ng/app/api/views.py` so to avoid duplicating it in multiple files and the need to import it, we added a static file.

Other solution would be having it **only** on `setup.py` and in other places use `pkg_resources.get_distribution("galaxy_ng").version` but that requires `galaxy_ng` package to be always installed.